src/multimediaaudioplayer.cc

```diff
@@ -20,13 +20,6 @@ MultimediaAudioPlayer::MultimediaAudioPlayer()
   player( 0, QMediaPlayer::StreamPlayback )
   #endif
 {
-  typedef void ( QMediaPlayer::*ErrorSignal )( QMediaPlayer::Error );
-  #if ( QT_VERSION < QT_VERSION_CHECK( 6, 0, 0 ) )
-  connect( &player,
-           static_cast< ErrorSignal >( &QMediaPlayer::error ),
-           this,
-           &MultimediaAudioPlayer::onMediaPlayerError );
-  #else
```

Comment:
Yet another crazy thing written by that person with purple avatar.

To avoid changing the slot's function signature, he alias a function pointer type, then cast the signal's function reference 😅.

I honestly don't know what is going on here. How can this even works. A realm of unknown.

Reply:
The code is for Qt5, so I just delete it.